OpenChange/iCalEvent+MAPIStore.m

```diff
@@ -457,7 +460,14 @@ - (void) updateFromMAPIProperties: (NSDictionary *) properties
                     {
                       [person setParticipationStatus: newPartStat];
                       [person setRsvp: @"TRUE"];
-                      [person setRole: @"REQ-PARTICIPANT"];
+                      if (i < ccFirstIndex)
```

Comment:
I want to make sure it follows the spec to avoid testing empirically with all supported Outlook version. A simple search in MS-OXOCAL Section 2.2.4.10.7 says the recipient type is 0x01 as required and 0x02 as optional. Searching for Recipient Type property in a Recipient Row shows, at least, two results were found MS-OXOMSG Section 2.2.3.1 and MS-OXCMSG 2.2.3.1.2 describing MAPI_TO as 0x01 and MAPI_CC as 0x02. So it is relevant to put this information in both places (commits and code documentation) to avoid the feeling of hack as the whole and exact information is in place.